src/services/ledger/accounts.ts

```diff
@@ -32,21 +32,22 @@ export const resolveAccountId = (accountPath: string | string[]) => {
     id = path[1]
   }
 
-  return id
+  // TODO: add check for WalletId syntax validity
```

Comment:
What does this mean `syntax validity`?

Reply:
should be `uuid-v4`. but not sure this is needed.

Reply:
For types that originate from our domain we should assume validity. The only place walletId's can be generated is internally so for this not to be valid here would be very strange.

Reply:
yup, removing the comment/type